Makefile

```diff
@@ -17,13 +17,7 @@ endif
     # Remove tekton-pipelines-resolvers-ns
 	rm -r charts/tekton-pipeline/templates/tekton-pipelines-resolvers-ns.yaml
 	# Amend subjects.namespace with release.namespace
-	yq -i '.subjects[].namespace = "{{ .Release.Namespace }}" '   charts/tekton-pipeline/templates/tekton-pipelines-resolvers-namespace-rbac-rb.yaml
-	yq -i '.subjects[].namespace = "{{ .Release.Namespace }}" '   charts/tekton-pipeline/templates/tekton-pipelines-resolvers-crb.yaml
-	yq -i '.subjects[].namespace = "{{ .Release.Namespace }}" '	  charts/tekton-pipeline/templates/tekton-pipelines-webhook-rb.yaml
-	yq -i '.subjects[].namespace = "{{ .Release.Namespace }}" '	  charts/tekton-pipeline/templates/tekton-pipelines-controller-rb.yaml
-	yq -i '.subjects[].namespace = "{{ .Release.Namespace }}" '	  charts/tekton-pipeline/templates/tekton-pipelines-controller-tenant-access-crb.yaml
-	yq -i '.subjects[].namespace = "{{ .Release.Namespace }}" '	  charts/tekton-pipeline/templates/tekton-pipelines-webhook-cluster-access-crb.yaml
-	yq -i '.subjects[].namespace = "{{ .Release.Namespace }}" '	  charts/tekton-pipeline/templates/tekton-pipelines-webhook-leaderelection-rb.yaml
+	find . -type f \( -name "*-crb*" -o -name "*-rb*" \) ! -name "*-role*" ! -name "*-info*" -exec yq -i '.subjects[].namespace = "{{ .Release.Namespace }}"' "{}" \;
```

Comment:
I think this would be more robust since it wouldn't add namespace where it isn't already set in any new resource either
```suggestion
	find . -type f \( -name "*-crb.yaml" -o -name "*-rb.yaml" \)  -exec yq -i '(.subjects[] | select(has("namespace"))).namespace = "{{ .Release.Namespace }}"' "{}" \;
```

Reply:
Yes I agree, thank you for reviewing. I like the way you have considered the select statement in the yq command to refine the replacement of the namespace value.

Reply:
I tested and commit your suggestion. Thanks :)